web/packages/teleport/src/Navigation/categories.ts

```diff
@@ -18,9 +18,10 @@
 
 export enum NavigationCategory {
   Resources = 'Resources',
-  Access = 'Access',
-  Identity = 'Identity',
-  Policy = 'Policy',
+  ZeroTrustAccess = 'Zero Trust Access',
+  MachineWorkloadID = 'Machine & Workload ID',
```

Comment:
```suggestion
  MachineWorkloadId = 'Machine & Workload ID',
```

This better matches our frontend naming conventions.